src/Waives.Http/WaivesClient.cs

```diff
@@ -79,23 +98,10 @@ public int Timeout
         /// <param name="clientId">The Waives API Client ID to use when authenticating with the service.</param>
         /// <param name="clientSecret">The Waives API Client Secret to use when authenticating with the
         /// service.</param>
-        public async Task Login(string clientId, string clientSecret)
+        public WaivesClient Login(string clientId, string clientSecret)
```

Comment:
I think this method could be removed, now, and instead provide the client Id and secret to the factory fn.

Reply:
Leaving this as-is allows us to support API deployments that have no authentication (e.g. local development).